recipe/meta.yaml

```diff
@@ -77,53 +80,77 @@ outputs:
         # so that one could run cmake for generating *.bc, and other
         # common data files.
         - arrow-cpp ={{ arrow_version }}=*cpu
-        - bisonpp
+        - bisonpp  # [linux64]
         - blosc
         - boost-cpp
         - clangdev {{ llvm_version }}
         - double-conversion
-        - flex
+        - flex  # [linux64]
         - fmt
         - geos
         - glog
         - llvmdev {{ llvm_version }}
         - llvm {{ llvm_version }}
         - libarchive
+        - libevent  # [win]
         - libgdal
         - libpng
         - librdkafka
+        - tbb
+        - tbb-devel
```

Comment:
Is tbb and tbb-devel needed just for windows?

Reply:
Yes! The linux build works just fine.